AndroidTest/app/src/main/java/cloudant/com/androidtest/keyprovider/ProviderTestUtil.java

```diff
@@ -0,0 +1,28 @@
+package cloudant.com.androidtest.keyprovider;
+
+import com.cloudant.sync.datastore.encryption.KeyData;
+
+import java.security.SecureRandom;
+import java.util.Random;
+import java.util.UUID;
+
+public class ProviderTestUtil {
+    public static String getUniqueIdentifier() {
+        return "test-id-" + UUID.randomUUID();
+    }
+
+    public static KeyData createKeyData() {
+        byte[] encryptedDPK = new byte[32];
+        byte[] salt = new byte[32];
+        byte[] iv = new byte[32];
+        int iterations = new Random().nextInt(100000);
+        String version = "1." + System.currentTimeMillis();
```

Comment:
Just chose a random int, as for iterations, and make your Random() object static to avoid duplication more certainly than millisecond clock time.